src/api/TradeAPI.ts

```diff
@@ -113,4 +134,130 @@ export default class TradeAPI {
       txOpts
     );
   }
+
+  /**
+   * Call 0x API to generate a trade quote for two SetToken components.
+   *
+   * @param  fromToken            Address of token being sold
+   * @param  toToken              Address of token being bought
+   * @param  fromTokenDecimals    Token decimals of token being sold (ex: 18)
+   * @param  toTokenDecimals      Token decimals of token being bought (ex: 18)
+   * @param  rawAmount            String quantity of token to sell (ex: "0.5")
+   * @param  fromAddress          SetToken address which holds the buy / sell components
+   * @param  setToken             SetTokenAPI instance
+   * @param  gasPrice             (Optional) gasPrice to calculate gas costs with (Default: fetched from GasNow)
+   * @param  slippagePercentage   (Optional) maximum slippage, determines min receive quantity. (Default: 2%)
```

Comment:
Did you grab these defaults from the service?

Reply:
Yes... from heroku, here: https://dashboard.heroku.com/apps/set-core-production/settings

![Screen Shot 2021-07-12 at 2 48 42 PM](https://user-images.githubusercontent.com/7332026/125360351-98fb5980-e320-11eb-9e4a-cdce7fa57222.png)

The whole number percentage is then divided by 100 before being passed to Zero Ex here: 
https://github.com/SetProtocol/set.js/blob/2c360f0f140a512e11c0ad0ddc9c6de6c9bc2caf/src/api/utils/tradeQuoter.ts#L231